src/components/publishing/read_more/read_more_wrapper.tsx

```diff
@@ -18,9 +18,8 @@ class ReadMoreWrapper extends React.Component<ReadMoreWrapperProps, ReadMoreWrap
       truncationHeight: "100%",
     }
   }
-
   componentDidMount() {
-    this.truncateArticle()
+    setTimeout(this.truncateArticle, 250)
```

Comment:
This is temporary -- this `componentDidMount` is getting fired before `ImageCollection`s have had a chance to check its size (with react-sizeme) and adjust image heights. I'm happy to remove this since it's ugly but this seems to work for now.